File: web/main/src/common/models/class-type.ts
export enum ClassType {
	Fighter = 1,
	Barbarian = 2,
	Rogue = 3,
	Ranger = 4,
	Monk = 5,
	Paladin = 6,
	Cleric = 7,
	Druid = 8,
	Wizard = 9,
	Sorcerer = 10,
	Bard = 11,
	Warlock = 12,
	Artificer = 13
}

export const ClassTypeNameMap: Record<ClassType, string> = {
	[ClassType.Fighter]: 'Fighter',
	[ClassType.Barbarian]: 'Barbarian',
	[ClassType.Rogue]: 'Rogue',
	[ClassType.Ranger]: 'Ranger',
	[ClassType.Monk]: 'Monk',
	[ClassType.Paladin]: 'Paladin',
	[ClassType.Cleric]: 'Cleric',
	[ClassType.Druid]: 'Druid',
	[ClassType.Wizard]: 'Wizard',
	[ClassType.Sorcerer]: 'Sorcerer',
	[ClassType.Bard]: 'Bard',
	[ClassType.Warlock]: 'Warlock',
	[ClassType.Artificer]: 'Artificer'
};

export const classTypeOptions = Object.values(ClassType)
	.filter((value) => typeof value !== 'string')
	.map((type) => ({
		value: type,
		label: ClassTypeNameMap[type]
	}));

File: api/SkeletonLabRpg/SkeletonLabRpg.Common/Database/Enums/ClassType.cs
using System.ComponentModel;

namespace SkeletonLabRpg.Common.Database.Enums;

public enum ClassType
{
    [Description("Fighter")]
    Fighter = 1,
    [Description("Barbarian")]
    Barbarian = 2,
    [Description("Rogue")]
    Rogue = 3,
    [Description("Ranger")]
    Ranger = 4,
    [Description("Monk")]
    Monk = 5,
    [Description("Paladin")]
    Paladin = 6,
    [Description("Cleric")]
    Cleric = 7,
    [Description("Druid")]
    Druid = 8,
    [Description("Wizard")]
    Wizard = 9,
    [Description("Sorcerer")]
    Sorcerer = 10,
    [Description("Bard")]
    Bard = 11,
    [Description("Warlock")]
    Warlock = 12,
    [Description("Artificer")]
    Artificer = 13,
}